### ISSUE TITLE
Grammar for p:run is incorrect

### ISSUE THREAD

**Issue by xatapult** [xproc/3.0-grammar#36]
The schema for p:run is incorrect. I think we should correct this.

**Comment by ndw** [comment]
Incorrect, or just missing?

**Comment by xatapult** [comment]
Incorrect. There is a content model for p:run but it doesn't follow the spec.

**Comment by ndw** [comment]
Moved to https://codeberg.org/XProc/specification/issues/96
